.github/workflows/release-pypi.yml

```diff
@@ -115,18 +111,21 @@ jobs:
         with open("versions.json", "w") as fw:
             json.dump(vers, fw)
       shell: python
-    - name: GIT Commit
+    - run: cat versions.json
+    - name: GIT commit & push
+      env:
+        BRANCH_NAME: "trigger/lightning-${{ env.TAG }}"
       run: |
+        git config --global user.name "PL Ghost"
+        git config --global user.email pl-github@grid.ai
+        git checkout -b ${BRANCH_NAME}
         git add versions.json
-        git commit -m "bumping lightning version -> ${TAG}"
-        cat versions.json
-    - name: GIT Push
-      run: |
         git status
-        git push
+        git commit -m "bumping lightning version -> ${TAG}"
+        git push -u origin ${BRANCH_NAME} -f
```

Comment:
Does the user pushing the commit have the write permission (probably admin-level is required to push to the main branch if the main branch is protected)?

Reply:
Oh it's changed to `trigger/lightning-` :) LGTM!